packages/data/src/onboarding/actions.js

```diff
@@ -217,6 +217,29 @@ export function* updateProfileItems( items ) {
 	}
 }
 
+/**
+ * Used to keep backwards compatibility with the extended task list filter on the client.
+ * This can be removed after version WC Admin 2.10 (see deprecated notice in resolvers.js).
+ *
+ * @param {Object} task the returned task object.
+ * @param {Array}  keys the keeps to keep in the task object.
```

Comment:
Nitpick, I think that there is a typo here.

Reply:
Thanks good catch :)